spring-integration-mqtt/src/main/java/org/springframework/integration/mqtt/inbound/MqttPahoMessageDrivenChannelAdapter.java

```diff
@@ -51,6 +54,25 @@
 public class MqttPahoMessageDrivenChannelAdapter extends AbstractMqttMessageDrivenChannelAdapter
 		implements MqttCallback, ApplicationEventPublisherAware {
 
+	/**
+	 * Acknowledgment mode.
+	 *
+	 * @since 5.3
+	 */
+	public enum AckMode {
+
+		/**
+		 * Client will auto ack when integration flow returns.
+		 */
+		AUTO,
+
+		/**
+		 * User must ack via the {@link IntegrationMessageHeaderAccessor#ACKNOWLEDGMENT_CALLBACK} header.
+		 */
+		MANUAL
```

Comment:
Can't we just live with similar to the client option like `manualAcks(boolean)` ?